client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -15,7 +15,7 @@ import { getCurrencyTabTitle } from './utils';
 import BalanceBlock from './balance-block';
 import BalanceTooltip from './balance-tooltip';
 import { documentationUrls, fundLabelStrings } from './strings';
-import InstantDepositButton from 'deposits/instant-deposits/button-and-modal';
+import InstantDepositButton from 'deposits/instant-deposits';
```

Comment:
The new version of `<InstantDepositButton />` that was contained in `instant-deposits/button-and-modal.js` has been moved to `instant-deposits/index.tsx`, which is being imported here.